### ISSUE TITLE
Viewing images

### ISSUE THREAD

**Issue by tgwoodcock** [marts/hdf5view#2]
Hi @marts, thanks again for this beautifully written and efficient library! I'm interested in viewing images in hdf5 files and saw that you've already made a great start in your imageview branch. This is something which would be very useful at work at the moment and so I've added some functionality in the imageview_dev branch on my fork. The image viewing seems to be working well and it's possible to have several image tabs open and also to navigate to other nodes on the table or other image tabs. If this might be of interest, I'd be happy to contribute this to your project (just thought I'd ask before starting a PR as there are more significant changes than the last one).

**Comment by marts** [comment]
Hi @tgwoodcock, I'd be more than happy for you to contribute. This project was inspired by some work I was doing with large datasets in HDF5 format (MEG data), however I wanted to create a more general purpose tool that would be useful for exploring any HDF5 data. Unfortunately, I no longer work in this area, so this project has fallen by the wayside...I'm not sure I currently have the setup to even run it. Therefore, any input you have is most welcome, maybe we could look at sharing/transfering ownership if that suits you...I'm really glad you are finding it useful.

**Comment by tgwoodcock** [comment]
Hi @marts, it's definitely useful and user friendly and the choice of libraries it's based on gives it several advantages over other hdf5 viewers I've tried. I think the image display is now at the point where it does everything I need it to do. I'd certainly be happy to make a PR for that. I could imagine contributing something about showing plots (rather than images) later but with a lower priority. We could even consider a release on PyPI to make it more accessible. If you've moved on from this topic we could consider some kind of shared ownership which would make it possible for me to handle some of these things. Maybe setting me as a collaborator on this repo could work? That way, your contributions are still recognised.

**Comment by marts** [comment]
@tgwoodcock I've added you as a collaborator. Go for it :)

**Comment by tgwoodcock** [comment]
Hi @marts, just wanted to check if you're ok with the way the authors and maintainers parameters in pyproject.toml appear in the latest update. I'll update the readme soon and will probably go for a test on TestPyPI.

**Comment by tgwoodcock** [comment]
Hi @marts, and we're up on PyPI now and I made a new release on github to reflect that. One thing I don't seem to be able to do as a collaborator is add topics to the github repo. If you have a moment, would you mind adding some topics, maybe similar to the keywords in pyproject.toml?

**Comment by marts** [comment]
@tgwoodcock done. Thanks again for all your hard work on this.

**Comment by tgwoodcock** [comment]
Thanks @marts. I guess we can probably close this issue for now.
